packages/gallery/src/components/item/videos/videoItem.js

```diff
@@ -22,10 +30,19 @@ class VideoItem extends React.Component {
     };
   }
 
+  playingFromIframeClick = false;
+  pausingFromIframeClick = false;
+
   componentDidMount() {
     this.dynamiclyImportVideoPlayers();
   }
 
+  isVimeoVideo = () =>
+    this.props.videoUrl && this.props.videoUrl.includes('vimeo.com');
```

Comment:
This code is duplicated. You can do it better

Reply:
sure